src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, onDelete }) => {
+  const handleDelete = () => {
+    onDelete(book.id);
+  };
+
+  return (
```

Comment:
- [ ] In your `/books` page should have rendered a list of books with the form to add or delete a book. Which functionalities isn't necessary to implement in this part of the project